platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/activity/BaseActivityTest.java

```diff
@@ -60,7 +53,6 @@ public void beforeTest() {
   }
 
   protected void validateTestSetup() {
-    Assert.assertTrue("Device is not connected to the Internet.", isConnected(rule.getActivity()));
```

Comment:
How about instead of asserting, we'll log the network state? This would give us more information about why the test might have failed. I like the changes regardless.